src/shared/components/variant-alleles-sequences/VariantAllelesSequences.scss

```diff
@@ -1,3 +1,7 @@
 .expanded span {
   display: block;
+}
+
+.alleleSequence {
```

Comment:
This whole component is about allele sequences 🙂 Let's try to think what the element with this class means. Is it, perhaps, the compact one-line representation of the allele sequences? `.compact`?

Reply:
or `.trimmedSequence` ?